package.json

```diff
@@ -42,6 +44,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "fastify": "^4.5.3",
     "husky": "^8.0.1",
+    "ioredis": "^5.2.3",
```

Comment:
only one doubt about this, which might be the reason why the official fastify plugin uses `any` instead. if we use a typing from a package which is only available in dev, what happens when the package is published?

Reply:
That's a good question I'll leave any then.

Reply:
Maybe we can do smth like this ? https://stackoverflow.com/questions/62047806/how-do-i-set-a-peer-dependency-optional